crates/swc_common/src/syntax_pos/hygiene.rs

```diff
@@ -138,85 +133,6 @@ impl Mark {
         Mark(raw)
     }
 
-    #[inline]
-    pub fn parent(self) -> Mark {
-        #[cfg(all(feature = "__plugin_mode", target_arch = "wasm32"))]
-        return Mark(unsafe { __mark_parent_proxy(self.0) });
-
-        #[cfg(not(all(feature = "__plugin_mode", target_arch = "wasm32")))]
-        return HygieneData::with(|data| data.marks[self.0 as usize].parent);
-    }
-
-    #[inline]
-    pub fn is_builtin(self) -> bool {
```

Comment:
Also, I search through the codebase, and find this function is never used, and also  `MarkData.is_builtin` field is never used too, so delete it.